examples/shared-variables/templates/styles/style1.js

```diff
@@ -0,0 +1,30 @@
+const sharedColors = require('../variables/colors');
+const { combineVariables } = require('../../../../dist/main');
+
+module.exports.context = {
+  sources: {
+    root: "Example"
+  },
+  colors: combineVariables(sharedColors, {
+    road: 'gray'
+  })
+};
```

Comment:
This is an example of how one would re-use common colors--define them in a separate file, import them, then combine them using `combineVariables()`. In this case, the style uses all of the `sharedColors` but overrides `road`.